app/views/application/_reviews.slim

```diff
@@ -49,7 +49,7 @@
                   -if review.recent?
                     | Current
                   -else
-                    ==link_to_revision(review.revision)
```

Comment:
is review.revision not already a revision object? (Same with below for row.revision)

Reply:
You're right. I had done this because I could not figure out why it was coming up as nothing because metadata and non-metadata revisions have the same commit_id. This has been fixed in the actual helper. Reverting this. Good catch.